metabase/metabase
# static viz measure text

Closes https://github.com/metabase/metabase/issues/24672

## Changes

Uses pre-calculated char widths of regular and bold Lato fonts to calculate text widths in static viz. For now, we render static viz only with Lato font but later I'll add more fonts.

## How to verify

- Ensure the internal static viz page looks as expected http://localhost:3000/_internal/static-viz
- Render various static charts by sending dashboard subscriptions and ensure it looks as expected
